app/Http/Controllers/ArticleController.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\ArticleStoreRequest;
+use App\Http\Requests\ArticleUpdateRequest;
+use App\Models\Article;
+use App\Models\Category;
+use App\Models\Tag;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Storage;
+use Inertia\Inertia;
+
+class ArticleController extends Controller
+{
+    /**
+     * @param \Illuminate\Http\Request $request
+     * @return \Illuminate\Http\Response
+     */
+    public function index(Request $request)
+    {
+        $articles = Article::latest()->get()->map(function (Article $article) {
```

Comment:
Why not use [Eloquent: API Resources](https://laravel.com/docs/8.x/eloquent-resources#introduction) here?